public/data_load/enhanced-table-request-handler.js

```diff
@@ -113,9 +114,13 @@ export async function enhancedTableRequestHandler ({
   // enrich response: total & aggs
   response.totalHits = _.get(searchSource, 'finalResponse.hits.total', -1);
   response.aggs = aggs;
-  response.columns.forEach(column => {
+  // Columns already have meta in Kibana 7.12
+/*   response.columns.forEach(column => {
     column.meta = serializeAggConfig(column.aggConfig);
-  });
+  }); */
+
+  //Add aggConfig to each column
+  response.columns = await enrichColumnsWithAggconfig(response.columns)
```

Comment:
instead of creating a new AggConfig object, you should get existing one in `response.aggs` object, like this:

``` javascript
  response.columns.forEach(column => {
    column.aggConfig = column.aggs.byId(column.meta.sourceParams.id);
  });
```

Reply:
Modified with your suggestion. Thanks!